docs/osx/01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/plugin-repo-overview.drawio.svg

```diff
@@ -1,351 +1,41 @@
                     PluginRepoRegistry
-                </text>
-            </switch>
-        </g>
-        <rect x="40" y="40" width="120" height="130" rx="18" ry="18" fill="rgb(255, 255, 255)" stroke="rgb(0, 0, 0)" pointer-events="all"/>
-        <rect x="30" y="30" width="40" height="20" rx="3" ry="3" fill="rgb(255, 255, 255)" stroke="#000000" pointer-events="all"/>
-        <g transform="translate(-0.5 -0.5)">
-            <switch>
-                <foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;">
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 40px; margin-left: 31px;">
-                        <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
-                            <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
-                                0x54...
-                            </div>
-                        </div>
-                    </div>
-                </foreignObject>
-                <text x="50" y="44" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                </text></switch><rect width="120" height="130" x="40" y="40" fill="#FFF" stroke="#000" pointer-events="all" rx="18" ry="18"/><rect width="40" height="20" x="30" y="30" fill="#FFF" stroke="#000" pointer-events="all" rx="3" ry="3"/><switch transform="translate(-.5 -.5)"><foreignObject width="100%" height="100%" pointer-events="none" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible;text-align:left"><div xmlns="http://www.w3.org/1999/xhtml" style="display:flex;align-items:unsafe center;justify-content:unsafe center;width:38px;height:1px;padding-top:40px;margin-left:31px"><div data-drawio-colors="color: rgb(0, 0, 0);" style="box-sizing:border-box;font-size:0;text-align:center"><div style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;pointer-events:all;white-space:normal;overflow-wrap:normal">0x54...</div></div></div></foreignObject><text x="50" y="44" font-family="Helvetica" font-size="12" text-anchor="middle">
```

Comment:
i only comment here thses auto or generated file should not be commited into PRs more than once

we need to find a different way to deal with them